content/post/Android_Root101.en.md

```diff
@@ -85,13 +88,12 @@ _After completing the process make sure to clear the Dalvik Cache and Cache memo
 
 _Reboot the Device._
 
-**_Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares._**
+{{<yellowborder>}}Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares.{{</yellowborder>}}
```

Comment:
Again some reference images needed.

Reply:
Added two images. Next blog me I'll add a lot of descriptive images. Pakka Promise! 😄

Reply:
You know he means adding images in the PR discussion for what this stuff actually looks like after rendering as we cannot guess everything from the code

He's not talking about adding images to the blog :rofl: